tower-layer/src/lib.rs

```diff
@@ -90,7 +90,7 @@ pub use self::{
 /// is also decoupled from client or server concerns. In other words, the same
 /// log middleware could be used in either a client or a server.
 ///
-/// [`Service`]: https://docs.rs/tower/trait.Service.html
+/// [`Service`]: https://docs.rs/tower/*/tower/trait.Service.html
```

Comment:
Same:

```suggestion
/// [`Service`]: tower_service::Service
```

Reply:
Thanks for reviewing. I tried but failed. Any suggestions?

```log
$ cargo doc --no-deps --all-features 
error: unresolved link to `tower_service::Service`
  --> tower-layer/src/lib.rs:17:18
   |
17 | //! [`Service`]: tower_service::Service
   |                  ^^^^^^^^^^^^^^^^^^^^^^ no item named `tower_service` in scope
   |
note: the lint level is defined here
  --> tower-layer/src/lib.rs:8:9
   |
8  | #![deny(broken_intra_doc_links)]
   |         ^^^^^^^^^^^^^^^^^^^^^^
   = note: `#[deny(rustdoc::broken_intra_doc_links)]` implied by `#[deny(broken_intra_doc_links)]`

error: aborting due to previous error

error: could not document `tower-layer`

warning: build failed, waiting for other jobs to finish...
error: build failed
```

Same error with `cargo doc --all-features`.